core/src/components/searchbar/searchbar.ionic.scss

```
@@ -1,46 +1,57 @@
-@use "searchbar.common";
 @use "../../themes/ionic/ionic.globals.scss" as globals;
+@use "searchbar.common";
+@use "./searchbar.ionic.vars" as searchbarVars;
```

```suggestion
@use "./searchbar.ionic.vars" as vars;
```

Can we rename this for consistency?

---

https://github.com/ionic-team/ionic-framework/pull/29917/commits/81fde1fd90380954797ca94a26212098b6300ec1 and https://github.com/ionic-team/ionic-framework/pull/29917/commits/8eb73311e59b8d57c7e169e4540fd873ae812f26